mautic/mautic
# Microsoft Dynamics FetchLeads

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? |N
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Required: )
#### Description:
This PR add the ability for users of Microsoft Dynamics users to fetch leads from the integration (today, only contacts and companies are fetchables).

#### Steps to test this PR:
1. Connect your integration (follow documentation)
2. In the feature tab, enable Leads
3. Create a lead in you MSD account
4. Launch command
5. See your lead in mautic contacts
6. Be happy !
